transformato/tests/test_restraints.py

```diff
@@ -10,136 +10,129 @@
+    ]  # Test proper selection and translation
 
     assert isinstance(testrestraint.force, simtk.openmm.CustomCentroidBondForce)
-
     assert isinstance(testrestraint.get_force(), simtk.openmm.CustomCentroidBondForce)
-
     assert isinstance(testrestraint_fb.force, simtk.openmm.CustomCentroidBondForce)
+    assert isinstance(
+        testrestraint_fb.get_force(), simtk.openmm.CustomCentroidBondForce
+    )
 
-    assert isinstance(testrestraint_fb.get_force(), simtk.openmm.CustomCentroidBondForce)
 
 @pytest.mark.restraints
 @pytest.mark.restraints_unittest
-def test_3DDistance():
-    assert(tfrs.get3DDistance(np.asarray([1,0,0]),np.asarray([0,0,0])))==1
+def test_3D_distance():
+    assert (tfrs.get3DDistance(np.asarray([1, 0, 0]), np.asarray([0, 0, 0]))) == 1
+
 
 @pytest.mark.restraints
 @pytest.mark.restraints_unittest
 def test_write_yaml(tmp_path):
-
-    class MockSystem():
+    
+    class MockSystem:
         def __init__(self):
-            self.tlc="LIG"
-            self.structure="structure2"
-    sys.modules["transformato.mutate"].cc_names_struc1=["C1","C2"]
-    sys.modules["transformato.mutate"].cc_names_struc2=["C1","C2"]
-    path=tmp_path/"test-config.yaml"
-    config=tfut.load_config_yaml(f"{TRAFO_DIR}/tests/config/test-2oj9-rsfe-restraints.yaml",".","./tmp/")
-    system=MockSystem()
-    current_step=4
-    tfrs.write_restraints_yaml(path,system,config,current_step)
+            self.tlc = "LIG"
+            self.structure = "structure2"
+
+    sys.modules["transformato.mutate"].cc_names_struc1 = ["C1", "C2"]
+    sys.modules["transformato.mutate"].cc_names_struc2 = ["C1", "C2"]
+    path = tmp_path / "test-config.yaml"
+    config = tfut.load_config_yaml(
+        f"{get_testsystems_dir()}/config/test-2oj9-rsfe-restraints.yaml", ".", "./tmp/"
+    )
+    system = MockSystem()
+    current_step = 4
+    tfrs.write_restraints_yaml(path, system, config, current_step)
 
     assert os.path.exists(path)
 
 
-
-
-
-
-
 @pytest.mark.restraints
 @pytest.mark.restraints_integrationtests
 def test_integration():
     """
     Full scale integration test of automatic and manual restraints, including an openMM test system.
-
     Essentially a modified openmm_run.py
-
     """
-    
 
     inputs = read_inputs(f"{PATH_2OJ9_DIR}step5_production.inp")
-    params = CharmmParameterSet(*(glob.glob(f"{PATH_2OJ9_DIR}../*/*.prm")+glob.glob(f"{PATH_2OJ9_DIR}../*/*.str")
-    +glob.glob(f"{PATH_2OJ9_DIR}../*/*.rtf")+glob.glob(f"{PATH_2OJ9_DIR}../toppar/*.prm")))
+    params = CharmmParameterSet(
+        *(
+            glob.glob(f"{PATH_2OJ9_DIR}../*/*.prm")
+            + glob.glob(f"{PATH_2OJ9_DIR}../*/*.str")
+            + glob.glob(f"{PATH_2OJ9_DIR}../*/*.rtf")
+            + glob.glob(f"{PATH_2OJ9_DIR}../toppar/*.prm")
+        )
+    )
     psf = CharmmPsfFile(f"{PATH_2OJ9_DIR}step3_input.psf")
     crd = read_crd(f"{PATH_2OJ9_DIR}step3_input.crd")
 
-    top=gen_box(psf,crd)
-    
+    top = gen_box(psf, crd)
+

```

Comment:
all of these functions/CLasses can be directly imported or defined as functions in the local scope. 
I think that would be preferable to importing functions from files.